src/pybind/mgr/dashboard/services/feedback.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+from typing import Optional
+from redminelib import Redmine
+from enum import Enum
+import errno
+import json
+import requests
+
+from requests.auth import AuthBase
+from mgr_module import CLICommand, CLIWriteCommand, CLIReadCommand, HandleCommandResult
+
+from .. import mgr
+from ..settings import Settings
+from ..exceptions import DashboardException, ScopeNotInRole
+from ..rest_client import RestClient
+from ..model.feedback import FeedbackModel
+
+
+class FeedbackProject(Enum):
+    dashboard = 46
+
+class FeedbackTracker(Enum):
+    bug = 1
+    feature = 2
+
+class FeedbackSeverity(Enum):
+    minor = '3 - Minor'
+    major = '2 - Major'
+    critical = '1 - Critical'
+
+class config:
+    url = 'tracker.ceph.com'
+    port = 443
+
+
+class RedmineAuth(AuthBase):
+    def __init__(self, access_key):
+        self.access_key = str(access_key)
+
+    def __call__(self, r):
+        r.headers['X-Redmine-API-Key'] = self.access_key
+        return r
+
+
+class CephTrackerClient(RestClient):
+    access_key = ''
+
+    def __init__(self):
+        super().__init__(config.url, config.port, client_name='CephTracker',
+                         ssl=True, auth=RedmineAuth(self.get_api_key()), ssl_verify=True)
+
+    @staticmethod
+    def get_api_key():
+        try:
+            access_key = Settings.CEPH_TRACKER_API_KEY
+        except KeyError as error:
+            raise DashboardException(msg='',
+                                     http_status_code=404,
+                                     component='')
+
+        return access_key
+
+    @RestClient.api_get('/issues/{issue_number}.json', resp_structure='*')
+    def get_issues(self, issue_number, request=None):
+        response = request()
+        return response
+
+    def create_issue(self, feedbackModel: FeedbackModel):
```

Comment:
No need to add the `model` thing to the instance. That's only for the class, the instance is a feedback instance.
```suggestion
    def create_issue(self, feedback: FeedbackModel):
```